src/reducers/ui.js

```diff
@@ -30,6 +32,10 @@ function update(state = State(), action: Action): Record<UIState> {
     case constants.SHOW_SOURCE: {
       return state.set("shownSource", action.sourceUrl);
     }
+
+    case constants.HIGHLIGHT_SOURCE: {
```

Comment:
I thought about doing that, but I'm sure which approach to take. @jasonLaster  Would you prefer making `SHOW_SOURCE` an array/object to having a separate functions for this feature?